react-charts/line-chart/step4.md

```diff
@@ -1,71 +1,34 @@
-Now we'll add multiple datasets to the line chart...
+Add tooltips to the line chart.
 
-## Task
+1) <strong>Make sure the `App.js` file is still open.</strong>
 
-1) Make sure the App.js file is still open
+2) <strong>Locate the `<Chart>` component.</strong>
 
-2) Locate the code that looks like the following:
+It will look like this:
 
 <pre class="file">
-&lt;ChartLine 
-  data={[
-    { name: &#39;Cats&#39;, x: &#39;2015&#39;, y: 1 }, 
-    { name: &#39;Cats&#39;, x: &#39;2016&#39;, y: 2 }, 
-    { name: &#39;Cats&#39;, x: &#39;2017&#39;, y: 5 }, 
-    { name: &#39;Cats&#39;, x: &#39;2018&#39;, y: 3 }
-  ]} 
-/&gt;
+&lt;Chart
+  maxDomain={{y: 10}}
+  minDomain={{y: 0}}
+  height={250}
+  width={600}
+&gt;
 </pre>
 
-3) Replace that section with the following code:
+3) <strong>Add a `constrainToVisibleArea` property inside of the `ChartVoronoiContainer`.</strong>

```

Comment:
Before this step need to add a containerComponent property to the Chart component like in area charts:

<img width="318" alt="Screen Shot 2019-12-06 at 11 36 51 AM" src="https://user-images.githubusercontent.com/20118816/70339391-c4e95b80-181c-11ea-9dde-e1bd42f12f4d.png">

Reply:
got it